aip/client-libraries/4236.md

```diff
@@ -0,0 +1,36 @@
+---
+id: 4236
+state: approved
+created: 2024-05-16
+---
+
+# Version-aware clients
+
+APIs may choose to annotate services with [google.api.api_version]. If 
+[google.api.api_version] is specified, version-aware clients **must** include
+the value of [google.api.api_version] in the request to the API. This allows
+services to abide by the schema and behavior of the service at the time that
+this API version was deployed. The format of the API version must be treated
```

Comment:
```suggestion
the value of [google.api.api_version] in the request to the API. The format of the API version must be treated
```

I'd move this sentence into a `Rationale` subheading below - this isn't guidance, it is reasoning